configure.ac

```diff
@@ -614,15 +614,7 @@ AC_ARG_ENABLE(
 #####################################################
 AM_CONDITIONAL([BUILD_PYTHON_LIBS], [test "x$enable_python_libs" = xyes])
 
-AS_IF([test "${enable_python_libs}" = "yes"],
-      [AM_PATH_PYTHON(2.6)
-       AS_IF([test -z $PYTHON], [PYTHON="python"])
-       PYTHON_NAME=`basename $PYTHON`
-       AC_CACHE_CHECK([for $PYTHON_NAME module: google.protobuf],
```

Comment:
Well, the goal of this patch would be to manage Python dependencies from the setup.py. This has  the advantage that you can easily set a version requirement on your dependency, and they are installed automatically from PyPi.